Makefile

```diff
@@ -0,0 +1,70 @@
+RUFF = poetry run ruff
+SRC = thalex tests examples
+RUFF_CFG = ./pyproject.toml
+
+
+.PHONY: clean
+clean: clean-build clean-pyc clean-test clean-docs
+
+.PHONY: clean-build
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	rm -fr deployments/build/
+	rm -fr deployments/Dockerfiles/open_aea/packages
+	rm -fr pip-wheel-metadata
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -fr {} +
+	find . -name '*.svn' -exec rm -fr {} +
+	find . -name '*.db' -exec rm -fr {} +
+	rm -fr .idea .history
+	rm -fr venv
+
+.PHONY: clean-docs
+clean-docs:
+	rm -fr site/
+
+.PHONY: clean-pyc
+clean-pyc:
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '__pycache__' -exec rm -fr {} +
+
+.PHONY: clean-test
+clean-test:
+	rm -fr .tox/
+	rm -f .coverage
+	find . -name ".coverage*" -not -name ".coveragerc" -exec rm -fr "{}" \;
+	rm -fr coverage.xml
+	rm -fr htmlcov/
+	rm -fr .hypothesis
+	rm -fr .pytest_cache
+	rm -fr .mypy_cache/
+	find . -name 'log.txt' -exec rm -fr {} +
+	find . -name 'log.*.txt' -exec rm -fr {} +
+
+.PHONY: tests
+tests:
+	poetry run pytest tests -vv --reruns 3 --reruns-delay 3
+
+fmt:
+	$(RUFF) --config $(RUFF_CFG) format $(SRC)
+	$(RUFF) --config $(RUFF_CFG) check $(SRC) --fix --no-cache
+
+lint:
+	$(RUFF) --config $(RUFF_CFG) format --check $(SRC)
+	$(RUFF) --config $(RUFF_CFG) check $(SRC) --no-fix --no-cache
+
+all: fmt lint tests
+
+test-docs:
+	echo making docs
+
+release:
+	$(eval current_version := $(shell poetry run tbump current-version))
+	@echo "Current version is $(current_version)"
+	$(eval new_version := $(shell python -c "import semver; print(semver.bump_patch('$(current_version)'))"))
+	@echo "New version is $(new_version)"
+	poetry run tbump $(new_version)
```

Comment:
Super helpful;

<img width="450" height="336" alt="image" src="https://github.com/user-attachments/assets/0975aaf8-013a-46ce-ba1d-709da23286d4" />


Creates a tagged release.

This triggers the workflow to build and push to pypi